makefile.shade

```diff
@@ -0,0 +1,28 @@
+
+var VersionPrefix = '1.2.0'
+var VERSION='0.1'
+var FULL_VERSION='0.1'
+use-standard-lifecycle
+k-standard-goals
+
+#pack-sources target='compile'
```

Comment:
....and just realized my response makes no sense here. Was thinking of an almost identical change I made here: https://github.com/aspnet/Common/pull/179.

Yes, your suggestion make senses. We should add a better convention for .Sources packages.